design-doc-template.adoc

```diff
@@ -82,73 +94,98 @@ __<List the installation types that are relevant for the features and remove any
 
 == Requirements
 
-__<Describe the requirements that must be fulfilled by this feature.>__
+////
+Describe the requirements that must be fulfilled by this feature.
 
-__<For analyses of a promotion of an existing feature to 'preview' or 'community' stability, only list new requirements;
+For analyses of a promotion of an existing feature to 'preview' or 'community' stability, only list new requirements;
 existing requirements from the feature being promoted are assumed to continue unless otherwise noted in the 'Changed
 requirements' section. Other analyses, including those for promotion to the 'default' stability level, must list all
-requirements.>__
+requirements.
+////
 
 === Changed requirements
 
-__<Only relevant for analyses of a promotion of an existing feature to 'preview' or 'community stability. Other analyses
-should remove this section.>__
+////
+Only relevant for analyses of a promotion of an existing feature to 'preview' or 'community stability. Other analyses
+should remove this section.
 
-__<For any existing requirements from the feature being promoted that are being changed or removed, describe the change.>__
+For any existing requirements from the feature being promoted that are being changed or removed, describe the change.
+////
 
 
 === Non-Requirements
 
-__<Use this section to explicitly discuss things that readers might think are required but which are not required.>__
+////
+Use this section to explicitly discuss things that readers might think are required but which are not required.
+////
 
 === Future Work
 
-__<Use this section to discuss requirements that are not addressed by this proposal but which may be addressed in later proposals.>__
+////
+Use this section to discuss requirements that are not addressed by this proposal but which may be addressed in later proposals.
+////
 
 == Backwards Compatibility
 
-__<Does this enhancement affect backwards compatibility with previously released versions of WildFly? Can the identified
-incompatibility be avoided?>__
+////
+Does this enhancement affect backwards compatibility with previously released versions of WildFly? Can the identified
+incompatibility be avoided?
+////
```

Comment:
Good adoc comment. Reviewers need reminders of what this area is meant to cover.